relay-cardinality/src/redis.rs

```diff
@@ -59,58 +164,41 @@ impl RedisSetLimiter {
             scope = &item_scope,
         );
 
-        let active_window = client.read_set(&self.to_redis_key(
-            organization_id,
-            &item_scope,
-            self.window.active_time_bucket(timestamp),
-        ))?;
-        let mut new_hashes = HashSet::new();
+        let keys = self
+            .window
+            .iter(timestamp)
+            .map(|time_bucket| self.to_redis_key(organization_id, &item_scope, time_bucket));
 
-        let mut new_cardinality = active_window.len();
-        let mut rejected = 0;
+        let hashes = entries.iter().map(|entry| entry.hash);
```

Comment:
Histogram for `entries.len()`?

Reply:
I would like that (unless we can already get that information indirectly).

Reply:
We kind of can get the average by dividing the total amount of hashes by the amount of set invocations.